app/src/main/java/com/example/cm/ui/adapters/MeetupRequestListAdapter.java

```diff
@@ -1,43 +1,45 @@
 package com.example.cm.ui.adapters;
 
 import android.content.Context;
+import android.os.Build;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.ImageView;
 import android.widget.TextView;
 
 import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
 import androidx.lifecycle.MutableLiveData;
 import androidx.recyclerview.widget.DiffUtil;
 import androidx.recyclerview.widget.RecyclerView;
 
 import com.example.cm.R;
 import com.example.cm.data.models.MeetupRequest;
 import com.example.cm.data.models.Request;
+import com.example.cm.data.models.User;
 import com.example.cm.databinding.ItemMeetupRequestBinding;
-import com.google.android.gms.maps.model.LatLng;
 import com.google.android.material.snackbar.Snackbar;
 
 import java.util.List;
 import java.util.Objects;
 
-import static com.example.cm.utils.Utils.convertToAddress;
-
 
 public class MeetupRequestListAdapter extends RecyclerView.Adapter<MeetupRequestListAdapter.MeetupRequestViewHolder> {
```

Comment:
Ändern sich Meetup-Daten oder der Name der Person, die dich einlädt, in der DB, wird hier nicht live geupdatet